paimon-spark/paimon-spark-common/src/main/java/org/apache/paimon/spark/SparkCatalog.java

```diff
@@ -103,6 +111,10 @@ public class SparkCatalog extends SparkBaseCatalog
 
     private String defaultDatabase;
 
+    public Set<TableCatalogCapability> capabilities() {
```

Comment:
Can SparkGenericCatalog support this capability?

Reply:
No, not in this PR.